data/korath/korath ships.txt

```diff
@@ -666,10 +667,11 @@ ship "Lor'kas Ik 577"
 	explode "large explosion" 40
 	explode "huge explosion" 50
 	"final explode" "final explosion large"
-	description "The largest of Korath production vessels are derived from a common and ancient chassis. This simple ship has been adapted for hauling materials between refinery and factory stations."
+	description "The largest of Korath production vessels are derived from a common and ancient chassis. Karek A'awoj are adapted to haul materials between refinery and factory stations. They're named after a productive colony of the Efreti, to honor their focus on peaceful industry."
```

Comment:
```suggestion
	description "The largest of Korath production vessels are derived from a common and ancient chassis. The simplest are adapted to haul materials between refinery and factory stations. Fittingly, they are named after a productive colony of the Efreti, to honor their focus on peaceful industry."
```

Each of the five derivative descriptions has a physical descriptor of some kind, which went missing here since I last reviewed.  Also, starting a sentence with "They're" felt far too... conversational?  Probably the biggest disagreement UnorderedSigh and I have is over how simple the English should be.